tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p><code>Note also that the remote backends require an open connection upon initialization. This would also be the case for QPUBackend.</code></p><p>I think the <code>how</code> is unwanted ;)   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
The tutorial looks good to me, I have just found one typo otherwise it's very clear !